tests/rptest/remote_scripts/python_librdkafka_serde_client.py

```diff
@@ -25,13 +25,15 @@
 from confluent_kafka.schema_registry import SchemaRegistryClient, topic_subject_name_strategy, record_subject_name_strategy, topic_record_subject_name_strategy
 from confluent_kafka.schema_registry.avro import AvroDeserializer, AvroSerializer
 from confluent_kafka.schema_registry.protobuf import ProtobufDeserializer, ProtobufSerializer
+from confluent_kafka.schema_registry.json_schema import JSONSerializer, JSONDeserializer
```

Comment:
note for reviewers: in this commit i mostly "filled the blanks" by looking at the Avro implementation